datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/licenses.py

```diff
@@ -160,29 +254,98 @@ async def get_data(url):
                 info['license'],
+                info['home_page'],
                 {extract_classifier_value(c) for c in info['classifiers'] if c.startswith('License ::')},
             )
 
 
 async def scrape_license_data(urls):
-    package_data = defaultdict(lambda: {'copyright': set(), 'licenses': [], 'classifiers': set()})
+    package_data = defaultdict(
+        lambda: {'copyright': dict(), 'licenses': set(), 'classifiers': set(), 'home_page': None, 'author': None}
+    )
 
     async with Pool() as pool:
-        async for package_name, package_copyright, package_license, license_classifiers in pool.map(get_data, urls):
+        async for package_name, package_author, package_license, home_page, license_classifiers in pool.map(
+            get_data, urls
+        ):
             data = package_data[package_name]
-            if package_copyright:
-                data['copyright'].add(package_copyright)
+            if package_author:
+                data['author'] = package_author
 
             data['classifiers'].update(license_classifiers)
             if package_license:
                 if ' :: ' in package_license:
                     data['classifiers'].add(extract_classifier_value(package_license))
                 else:
-                    data['licenses'].append(package_license)
+                    data['licenses'].add(package_license)
+
+            if home_page:
+                data['home_page'] = home_page
 
     return package_data
 
 
+def update_copyrights(package_name, license_id, data, ctx):
+    home_page = data['home_page']
+    repo_url = PACKAGE_REPO_OVERRIDES.get(package_name) if PACKAGE_REPO_OVERRIDES.get(package_name) else home_page
+    if repo_url:
+        cp, created_date = scrape_copyright_data(repo_url, ctx)
+        if cp:
+            data['copyright'][license_id] = cp
+    if data['author'] and not data['copyright'].get(license_id):
+        cp = 'Copyright {}{}'.format(created_date, data['author'])
+        if license_id in COPYRIGHT_ATTR_TEMPLATES:
+            cp = COPYRIGHT_ATTR_TEMPLATES[license_id].format(
+                year=created_date, author=data['author'], package_name=package_name, license=license_id
+            )
+        data['copyright'][license_id] = cp
+
+
+def probe_github(url, ctx):
+    if url.endswith('/'):
+        url = url[:-1]
+    owner_repo = re.sub(r'.*github.com/', '', url)
+    repo_api_url = f'https://api.github.com/repos/{owner_repo}'
+    repo_res = requests.get(repo_api_url, auth=get_auth_info(ctx.obj)).json()
+    def_branch = repo_res.get('default_branch')
+    created_date = repo_res.get('created_at') or ''
+    if created_date:
+        created_date = created_date[:4] + ' '
+    path = f'https://raw.githubusercontent.com/{owner_repo}/{def_branch}'
+    return path, created_date
+
+
+def scrape_copyright_data(url_path, ctx):
+    license_path, created_date = probe_github(url_path, ctx)
+    path = deepcopy(license_path)
```

Comment:
It looks like path is just a string

```suggestion
    path = license_path
```